src/Validation.hs

```diff
@@ -858,7 +858,7 @@ instance (NoValidationMonadError, Semigroup e) => Monad (Validation e) where
 type family NoValidationMonadError :: Constraint where
     NoValidationMonadError = TypeError
         ( 'Text "Type 'Validation' doesn't have lawful 'Monad' instance"
-        ':$$: 'Text "which means that you can't use 'Monad' methods with 'Validation'."
+        ' :$$: 'Text "which means that you can't use 'Monad' methods with 'Validation'."
```

Comment:
This was required because the parser for `ghc-9.6` **insists** on whitespace around operators.